src/axom/multimat/docs/sphinx/multimat_materials.rst

```diff
@@ -0,0 +1,120 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Materials
+******************************************************
+
+MultiMat defines materials over the cells in a mesh. This section describes how to
+use the MultiMat object to define materials.
+
+#######################
+Cell Material Relation
+#######################
+
+The distribution of materials over the mesh is determined using the **Cell-Material Relation** *(CMR)*.
+If a mesh has *N* cells and *M* materials then each cell can have *M* possible values
+if it contains all of the possible materials. There is a total of N*M values if each
+cell contains all materials. There are multiple ways to specify the CMR to MultiMat. A
```

Comment:
Suggestion: "...and *M* materials, each cell can have up to *M* possible values.  This means the mesh will hold N*M values if each cell contains all materials."

Reply:
Changed.

Reply:
Thanks for taking a look!